src/field/stake-out/index.md

```diff
@@ -0,0 +1,26 @@
+# How to Stake Out Points
+<Badge text="since Input 1.3.0" type="tip"/>
+
+Points in your survey layers can be staked out. Input will navigate you towards the selected point showing you the direction and distance.
+
+There are two stake out modes:
+- long navigation mode
+- short navigation mode
+
+:::tip
+Accurate stake out in short navigation mode may require an [external GPS receiver](../external_gps/) with GPS corrections.
+:::
+
+
+1. Select a point in your map window. From the form select the **Stake out** icon:
+![stake out](./stake-out.png)
+
+2. Stake out panel will appear showing the current distance and a connecting line between your position and the point.
+![long range](./stake-out-long.png)
+
+3. Once you are closer to the point, the stake out panel will switch to short navigation mode.
```

Comment:
Short mode is visible if user is less than 1m from destination point, but I am not sure if we want to mention that.